code/client/src/pages/Show/Swap.jsx

```diff
@@ -223,8 +223,10 @@ const Swap = ({ address }) => {
         dispatch(walletActions.fetchTokenBalance({ address, tokenType, tokenId, contractAddress, key }))
       }
     })
+    const filteredTrackedTokens = [...new Map(trackedTokens.map(v => [v.address, v])).values()]
```

Comment:
Could use lodash uniqBy

Reply:
Yep, the project has lodash, I will change it later. :)

Reply:
I changed it in master